docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -0,0 +1,289 @@
+.. _getting-started-hopsworks-ai:
```

Comment:
All the guide assume that the users are using a linux console for the azure CLI, what if they are on windows?

Reply:
The CLI is also available for Windows and I linked the guides in the install the azure cli link